synfig-core/src/synfig/time.h

```diff
@@ -70,7 +70,7 @@ class Time
 private:
 	value_type value_;
 
-	static value_type epsilon_() { return static_cast<value_type>(0.0005); }
+	static const value_type epsilon_;
```

Comment:
constexpr?

Reply:
It didn't work for ubuntu 20.
Thwre is a difference for constexpr in static members on c++14 and c++17.
So I changed to classic `const`.